README.md

```diff
@@ -19,7 +19,7 @@ PromptingTools.jl is not meant for building large-scale systems. It's meant to b
 
 Getting started with PromptingTools.jl is as easy as importing the package and using the `@ai_str` macro for your questions.
 
-Note: You will need to set your OpenAI API key as an environment variable before using PromptingTools.jl (see the [Creating OpenAI API Key](#creating-openai-api-key) section below). 
+Note: You will need to set your OpenAI API key as an environment variable before using PromptingTools.jl (see the [Creating OpenAI API Key](#creating-openai-api-key) section below). Note also that your OpenAI account has to be funded to access the OpenAI API.
```

Comment:
```suggestion
Note: You will need to set your OpenAI API key as an environment variable before using PromptingTools.jl (see the [Creating OpenAI API Key](#creating-openai-api-key) section below).

Following the introduction of [Prepaid Billing](https://help.openai.com/en/articles/8264644-what-is-prepaid-billing), you'll need to buy some credits to get started ($5 minimum).
```

Reply:
Let's add more detail and some references.

Reply:
I am fine with this suggestion, so I will commit it. Not sure what you mean by "more detail and some references", unless you refer to the subsection for this error in the FAQ. In which case, it has been added ^_^